paywall/src/hooks/browser/useListenForPostMessage.js

```diff
@@ -2,21 +2,32 @@ import { useEffect, useState } from 'react'
 import useConfig from '../utils/useConfig'
 import { getRouteFromWindow } from '../../utils/routes'
 
-export default function useListenForPostmessage(window) {
+export default function useListenForPostmessage(
```

Comment:
Do you mind adding JS doc comments to tell us a bit more about that this hook does?

Reply:
done

Reply:
wow, great job!

Reply:
Nit: do you mind using camelCase for this? (I see the previous code was indeed using the wrong case, but let's fix it here! Thanks!